app/controllers/cross_disciplinary_reports_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Provides the various CSV reports
+class CrossDisciplinaryReportsController < ApplicationController
+  def show
+    data = generator.generate(params.slice(:concept_uri))
```

Comment:
Do we need a separate controller for this, or can we just use the ReportsController and add `concept_uri` to the `slice()` there?

Reply:
Done.